src/univariate/continuous/ksdist.jl

```diff
@@ -120,16 +120,16 @@ function ceil_rems_mult(n,x)
 end
 
 # n!*(e/n)^n
-function stirling(n)    
+function stirling(n)
     if n < 500
-        s = 1.0
+        s = 1
```

Comment:
Yes. I thought there might be more issue in the function and therefore that using a different implementation would solve it but this issue should be easy to fix.